src/View/Form/ArrayContext.php

```diff
@@ -80,9 +73,9 @@ class ArrayContext implements ContextInterface
      * @param \Cake\Network\Request $request The request object.
      * @param array $context Context info.
      */
-    public function __construct(Request $request, array $context)
+    public function __construct(Request $request, array $context, $requestType)
```

Comment:
I can add a default value for the $requestType parameter but I thought the class is only used by the FormHelper internally.
Shall I still add the default value?